### PR TITLE
bump holo-nixpkgs to an unmerged branch

### PR DESCRIPTION
**Pull request by steveej**

### CODE REVIEW

File: test/clippy.bats [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @test "clippy version" {
  result="$( cargo clippy --version )"
  echo $result
- [[ "$result" == *0.1.51* ]]
+ [[ "$result" == *0.1.52* ]]
```
Comment (freesig) [inline test/clippy.bats]: Should this be 1.53?

**Review by freesig — APPROVED** [review]
Just one question

File: test/clippy.bats (hunk 1 above)
Comment (steveej) [reply test/clippy.bats]: it seems not, otherwise the test would fail. apparantly the versions don't strictly follow the rust toolchain version?

Comment (steveej) [reply test/clippy.bats]: checking this manually I'm confused:

```
$ nix-shell --run "cargo clippy --version"
clippy 0.1.53 (53cb7b0 2021-06-17
```

Comment (steveej) [reply test/clippy.bats]: I found the bug that was causing it to not fail on mismatch. thanks for leading me there :+1:

**Pull request merged**